src/aspire/volume/volume.py

```diff
@@ -461,48 +464,57 @@ def flip(self, axis=-3):
 
         return self.__class__(np.flip(self._data, axis), symmetry_group=symmetry)
 
-    def downsample(self, ds_res, mask=None):
+    def downsample(self, ds_res, mask=None, zero_nyquist=True):
         """
         Downsample each volume to a desired resolution (only cubic supported).
 
         :param ds_res: Desired resolution.
+        :param zero_nyquist: Option to keep or remove Nyquist frequency for even resolution.
+            Defaults to zero_nyquist=True, removing the Nyquist frequency.
         :param mask: Optional NumPy array mask to multiply in Fourier space.
         """
 
         original_stack_shape = self.stack_shape
         v = self.stack_reshape(-1)
 
         # take 3D Fourier transform of each volume in the stack
-        fx = fft.fftshift(fft.fftn(xp.asarray(v._data), axes=(1, 2, 3)))
+        fx = fft.centered_fftn(xp.asarray(v._data))
 
         # crop each volume to the desired resolution in frequency space
         fx = crop_pad_3d(fx, ds_res)
 
+        # If downsample resolution is even, optionally zero out the nyquist frequency.
+        if ds_res % 2 == 0 and zero_nyquist is True:
+            fx[:, 0, :, :] = 0
+            fx[:, :, 0, :] = 0
+            fx[:, :, :, 0] = 0
+
         # Optionally apply mask
         if mask is not None:
             fx = fx * xp.asarray(mask)
 
         # inverse Fourier transform of each volume
-        out = fft.ifftn(fft.ifftshift(fx), axes=(1, 2, 3)).real
+        out = fft.centered_ifftn(fx)
         out = out.real * (ds_res**3 / self.resolution**3)
 
         # returns a new Volume object
         return self.__class__(
-            xp.asnumpy(out), symmetry_group=self.symmetry_group
+            xp.asnumpy(out),
+            symmetry_group=self.symmetry_group,
         ).stack_reshape(original_stack_shape)
 
     def shift(self):
         raise NotImplementedError
 
-    def rotate(self, rot_matrices, zero_nyquist=True):
+    def rotate(self, rot_matrices, zero_nyquist=False):
```

Comment:
I don't recall why we decided to have a different default for `rotate` compared to the others. Can you remind me?